internal/astnav/tokens_test.go

```diff
@@ -20,7 +20,6 @@ import (
 )
 
 var testFiles = []string{
-	// !!! EOFToken JSDoc parsing is missing
 	// filepath.Join(repo.TestDataPath, "fixtures/astnav/eofJSDoc.ts"),
```

Comment:
even though I brought back `EndOfFileToken`, I'm not sure what to do here - this commented out file doesnt exist and I can't find its equivalent in Strada either